client/lib/graph/GraphProvider.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes, Children } from 'react';
+
+export default class GraphProvider extends Component {
+	static propTypes = {
+		graph: PropTypes.object.isRequired,
+		children: PropTypes.element.isRequired
+	};
+
+	static childContextTypes = {
+		graph: PropTypes.object.isRequired
+	};
+
+	getChildContext() {
+		return { graph: this.graph };
+	}
+
+	constructor( props, context ) {
+		super( props, context );
+		this.graph = props.graph;
```

Comment:
Yes, I have not thought a lot when writing this one :) I just copied Redux React Provider. So can't tell why. I think it may be useless :)